src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -185,16 +196,25 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
         }}
         text="Test Mint Tokens"
       />
-      {sortedActionsData.length ? (
+      {actions.length ? (
         <>
-          <ActionsListHeading onSortChange={setSortOption} />
-          <ActionsList items={sortedActionsData} />
-          {/* {loadMoreItems && (
+          <ActionsListHeading
+            sortDirection={sortDirection}
+            onSortChange={setSortDirection}
+          />
+          <ActionsList items={actions} />
+          {hasMoreItems && (
             <LoadMoreButton
-              onClick={showMoreItems}
+              onClick={() =>
```

Comment:
If you were to move the above into a hook, this could be returned from it as, e.g. "handleLoadMore"

Reply:
I agree with this